.github/workflows/runtests.yml

```diff
@@ -41,6 +41,12 @@ jobs:
       - name: Test all modules are listed in setup.py
         run: bin/test_setup.py
 
+      - run: sudo apt install xorg-dev libglu1-mesa libgl1-mesa-dev xvfb libxinerama1 libxcursor1
+      - run: pip install slotscheck pyglet . 
+
+      - name: Check for incorrect use of ``__slots__`` using slotscheck
+        run: xvfb-run -a -s "-screen 0 1400x900x24 +extension RANDR" -- python -m slotscheck --exclude-modules "(sympy.parsing.autolev._antlr.*|sympy.parsing.latex._antlr.*|sympy.galgebra)" sympy
```

Comment:
Is all this extra stuff here because of pyglet? Let's just exclude pyglet.

Reply:
Whatever this is for it should be removed.

Overly complicated things in CI are problematic and the slotscheck is not even remotely useful enough to warrant any kind of CI complexity.

Reply:
This is indeed for pyglet. Should I set up a new PR for the change?

Reply:
Yes, this PR was already merged so you will need to make a new PR.

Reply:
By the way, I opened https://github.com/sympy/sympy/issues/25891 to discuss pyglet generally. I'm a little surprised that it's still there. I'm not sure if it's really something worth keeping around, especially that it can add complexity like this.

Reply:
#25892 has been opened for the fix.

Reply:
I'm curious why this was even needed. Does slotscheck do anything other than import the modules? What does slotscheck do with pygletplot if you don't run it through xvfb-run like this?

Reply:
Importing pyglet requires an X-server, it seems. Without xvfb-run, slotscheck was throwing a NoSuchDisplayException whenever it tried to import pygletplot.